source/fuzz/fuzzer_pass_vector_dynamic.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2020 André Perez Maselco
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SOURCE_FUZZ_FUZZER_PASS_VECTOR_DYNAMIC_H_
+#define SOURCE_FUZZ_FUZZER_PASS_VECTOR_DYNAMIC_H_
+
+#include "source/fuzz/fuzzer_pass.h"
+
+namespace spvtools {
+namespace fuzz {
+
+// Looks for OpCompositeExtract and OpCompositeInsert instructions, checks if
+// the composite operand is a vector, creates constants for their literal
+// operands and randomly applies the transformation.
```

Comment:
I would abbreviate a bit: "Looks for OpCompositeExtract/Inser instructions on vectors, and replaces them with OpVectorExtract/InsertDynamic."  The stuff about adding constants is more of an implementation detail.

Reply:
Done.